manifests/04-rbac-rolebinding.yaml

```diff
@@ -79,3 +79,18 @@ subjects:
   - kind: ServiceAccount
     name: console-operator
     namespace: openshift-console-operator
+---
+kind: RoleBinding
```

Comment:
This resolves the RBAC issues when we re-enable the authentication in `05-config.yaml` below.